src/routes/app.ts

```diff
@@ -104,7 +104,7 @@ export default async function sessionCache(fastify: FastifyInstance) {
       path: '/',
       secure: true,
     });
-
+    
```

Comment:
can you revert these white space changes?  They just add clutter to the PR

Reply:
this file `app.ts` should not be in this PR because you just made white space changes - please revert

Reply:
can you revert these white space changes?